docs/serverless-v2/06-01-function-cr.md

```diff
@@ -11,16 +11,14 @@ kubectl get crd functions.serverless.kyma-project.io -o yaml
 
 ## Sample custom resource
 
-The following Function object creates a lambda which runs on the Node.js 8 runtime and responds to HTTP requests with "Hello John".
+The following Function object creates a lambda which responds to HTTP requests with "Hello John."
```

Comment:
`Hello John`. 
Without dot according to code

Reply:
yup, that is just AMe standard - dots in inverted commas, not outside :)